addons/website_event_track/static/src/css/website_event_track.css

```diff
@@ -1,32 +1,25 @@
-.o_wevent_event .ribbon-wrapper {
-  width: 60px;
-  height: 60px;
-  z-index: 5;
+.o_wevent_event .o_wevent_sponsor {
   overflow: hidden;
-  position: absolute;
-  top: 0;
-  right: 0;
+  position: relative;
 }
 
-.o_wevent_event .ribbon {
-  font: bold 13px Sans-Serif;
+.o_wevent_event .o_ribbon {
+  font: 12px Sans-Serif;
   color: #404040;
-  text-align: center;
-  -webkit-transform: rotate(45deg);
-  -moz-transform: rotate(45deg);
-  -ms-transform: rotate(45deg);
-  -o-transform: rotate(45deg);
-  position: relative;
-  padding: 5px 0;
-  left: -4px;
-  top: 8px;
-  width: 90px;
+  padding-top: 2px;
+  padding-bottom: 2px;
   -webkit-box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.3);
   -moz-box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.3);
   box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.3);
+
+  /* Move ribbon to top right corner */
+  right: 0;
+  left: unset;
+  transform-origin: top right;
+  transform: translateX(calc(-0.708 * (100% - 200px))) rotate(45deg) translateX(calc(100% - 100px));
```

Comment:
I'd rather use two classes per ribbon (o_ribbon + o_ribbon_left/o_ribbon_right) so that I can use the common class to determine if something has a ribbon at all, if you don't mind. And apparently, there are ribbons on both the left and the right in @Cocographique 's prototype for the new shop page so it may not be a bad thing to have both available in website.

Reply:
Let's go with that then yes